packages/payment/integration-tests/__tests__/services/payment-module/index.spec.ts

```diff
@@ -9,14 +12,134 @@ import {
   createPayments,
 } from "../../../__fixtures__"
 import { getInitModuleConfig } from "../../../utils/get-init-module-config"
-import { initModules } from "medusa-test-utils"
-import { Modules } from "@medusajs/modules-sdk"
 
 jest.setTimeout(30000)
 
 describe("Payment Module Service", () => {
   let service: IPaymentModuleService
 
+  describe("Payment Module Flow", () => {
+    let repositoryManager: SqlEntityManager
+    let shutdownFunc: () => Promise<void>
+
+    beforeAll(async () => {
+      const initModulesConfig = getInitModuleConfig()
+
+      const { medusaApp, shutdown } = await initModules(initModulesConfig)
+
+      service = medusaApp.modules[Modules.PAYMENT]
+
+      shutdownFunc = shutdown
+    })
+
+    afterAll(async () => {
+      await shutdownFunc()
+    })
+
+    beforeEach(async () => {
+      await MikroOrmWrapper.setupDatabase()
+      repositoryManager = await MikroOrmWrapper.forkManager()
+
+      service = await initialize({
```

Comment:
suggestion: you might also take the time to remove the initialize function and maybe you can apply the new migration generator in the local module config